addon/Config.jsm

```diff
@@ -11,11 +11,14 @@
 var EXPORTED_SYMBOLS = ["config"];
 
 var config = {
+  // TODO glind: Update notification message as needed
+  notificationMessage: "Please help us improve Firefox and the Web.",
```

Comment:
this is going to be part of the settable stuff, and part of the variations array.